scrapers/Assylum.yml

```diff
@@ -0,0 +1,36 @@
+name: assylum
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.assylum.com/session/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h3[@class='mas_title']/text()
+      Performers:
+        Name: 
+          selector: //span[@class='lc_info mas_description']/text()[1]
+          postProcess:
+            - replace:
+              - regex: (.+)\,
+                with: $1
```

Comment:
Scenes like this one `https://www.assylum.com/session/Noemie_Bilas/ass_eating_domination/603` the Performer name I got was `Noemie Bilas, Day 1` which could be fixed with some more regex.

But scenes like this `https://www.assylum.com/session/Leah-Winters/Sadistic-Sustenance/Rough-anal-domination-and-extreme-ass-fucking/590` don't have the performer names in the title. Not sure there is anything that can be done about that as I couldn't find any reliable way to get a performer name here.